core/domain/rights_manager.py

```diff
@@ -765,18 +766,19 @@ def check_can_voiceover_activity(user, activity_rights):
     if role_services.ACTION_EDIT_OWNED_ACTIVITY not in user.actions:
         return False
 
-    if (activity_rights.is_owner(user.user_id) or
+    if (
+            activity_rights.is_owner(user.user_id) or
             activity_rights.is_editor(user.user_id) or
             activity_rights.is_voice_artist(user.user_id)):
         return True
 
-    if (activity_rights.community_owned or
+    if (
```

Comment:
Can we please move line 776 here and move the angular bracket at the start of line 777 to on line 745? 

(Ditto here and else where) 

In general, if a comment is given for a particular line, please replicate it over all the changes.